modules/stochastic_tools/doc/content/source/samplers/ParallelSubsetSimulation.md

```diff
@@ -105,6 +105,14 @@ chains, the `AdaptiveMonteCarloDecision` class
 is used for decision-making. Please refer to [AdaptiveMonteCarloDecision](AdaptiveMonteCarloDecision.md)
 for more details.
 
+If the parameter `use_absolute_value = true`, the absolute value of `output_reporter` will be maximized. 
```

Comment:
Please use the `[!param](/Samplers/ParallelSubsetSimulation/param_name)` syntax for all referenced parameters